packages/neuron-ui/src/components/DataSetting/hooks.ts

```diff
@@ -52,7 +52,7 @@ export const useDataPath = (
     })
   }, [closeDialog, type])
   const onConfirm = useCallback(
-    e => {
+    (e: any) => {
       const { dataset } = e.currentTarget
```

Comment:
Why use any to declare `e`?

Reply:
> Why use any to declare `e`?

Cause some of the parameters are vaguely defined, such as `document.addEventListener('click', e =>{ ... })`, I'll make a double check to remove the `any` type as much as possible.